### DISCUSSION TITLE
Mapping buttons without Raspberry Pi OS?

### DISCUSSION THREAD

**Discussion by IRRDC** [CoolerMasterTechnology/Pi-Tool#11] [Q&A]
Can the buttons of the case be mapped if I'm not using Raspberry Pi OS? I want to use the Pi Case 40 for my XBIAN streaming box and would like to utilize the button (shutdown would be enough).

**Comment by mntns** [comment]
While the Pi Tool was designed for Raspberry Pi OS, I spontaneously don't see why it wouldn't work on XBIAN. It would have to be run in a regular desktop/window manager and not Kodi (or whatever frontend XBIAN uses per default) and currently (https://github.com/CoolerMasterTechnology/Pi-Tool/issues/7) the button mappings aren't persisted across restarts, so the Pi Tool would have to be run at the start each time. 

Besides that, you can get the latest Debian package release [here](https://github.com/CoolerMasterTechnology/Pi-Tool/releases/tag/v0.3.1).

**Reply by IRRDC** [reply]
As far as I know XBIAN doesn't come with any other window manager than Kodi. I was hoping for something like a start up script that I could add to the machine. As said, I don't need to customize anything or overclock the system, I just want to make the button usable to send a shutdown command to the system.
